src/sql/base/browser/dom.ts

```diff
@@ -118,3 +118,10 @@ export function trapKeyboardNavigation(container: HTMLElement): IDisposable {
 		}
 	});
 }
+
+/**
+ * Convert the SlickGrid's keydown event to VSCode standard keyboard event.
+ */
+export function convertJQueryKeyDownEvent(e: DOMEvent): StandardKeyboardEvent {
+	return new StandardKeyboardEvent((<any>e as JQuery.KeyDownEvent).originalEvent);
```

Comment:
Why not just have the parameter be a JQuery.KeyDownEvent to begin with? Casting like this is potentially risky...

Reply:
it is defined like that in the slickgrid's typing, maybe I also modify it.

Reply:
it is defined as a generic event: https://github.com/microsoft/azuredatastudio/blob/main/src/typings/slickgrid.d.ts#L65

Reply:
Bleh, oh well guess it's easy enough to verify we're using it only for keyboard events so guess we'll live with this.